services/mock-blockchain/src/server.ts

```
@@ -0,0 +1,199 @@
+    this.reputationContract = new MockReputationContract(config);
+    
+    this.setupMiddleware();
+    this.setupRoutes();
+  }
+
+  private setupMiddleware() {
+    this.app.use(cors());
+    this.app.use(bodyParser.json());
+    
+    // Error handling middleware
+    this.app.use((err: any, req: express.Request, res: express.Response, next: express.NextFunction) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)